pennylane/devices/experimental/default_qubit_2.py

```diff
@@ -18,13 +18,15 @@
 from typing import Union, Callable, Tuple, Optional, Sequence
 
 from pennylane.tape import QuantumTape, QuantumScript
+from pennylane.typing import Result, ResultBatch
 
 from . import Device
 from .execution_config import ExecutionConfig, DefaultExecutionConfig
 from ..qubit.simulate import simulate
 from ..qubit.preprocess import preprocess, validate_and_expand_adjoint
 from ..qubit.adjoint_jacobian import adjoint_jacobian
 
+Result_or_Batch = Union[Result, ResultBatch]
```

Comment:
```suggestion
Result_or_ResultBatch = Union[Result, ResultBatch]
```

Longer but more clear imo